src/components/Path/PathStage.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { IconSVG } from '../../.';
+
+export const PathStage = (props) => {
+  const { className, assistiveText, label, complete, current, selected, onStageClick, ...rest, } = props;
+  const sldsClasses = [
+    'slds-tabs_path__item',
+    { 'slds-is-complete': complete },
+    { 'slds-is-current': current },
+    { 'slds-is-incomplete': !complete && !current },
+    { 'slds-is-active': selected && !current },
+    className,
+  ];
+
+  return (
+    <li className={cx(sldsClasses)} role="presentation" {...rest}>
+      <a
+        aria-selected={selected}
+        className="slds-tabs_path__link"
+        role="option"
+        tabIndex={selected ? 0 : -1}
```

Comment:
`-1` removes keyboard access. via tabbing, is this intentional?

Reply:
yes: https://www.lightningdesignsystem.com/components/path/